doc/docs.md

```diff
@@ -813,6 +813,24 @@ A match statement can also be used to branch on the variants of an `enum`
 by using the shorthand `.variant_here` syntax. An `else` branch is not allowed
 when all the branches are exhaustive.
 
+```v
+c := `v`
+typ := match c {
+    `0`...`9` { 'digit' }
+    `A`...`Z` { 'uppercase' }
+    `a`...`z` { 'lowercase' }
+    else      { 'other' }
+}
+println(typ) // 'lowercase'
+```
+
+You can also use ranges as `match` conditions. If the value falls within the range
+of a branch, that branch will be executed.
```

Comment:
Please make clear that only the first matching branch is executed.

Reply:
```v
match x {
    0...10 {}
    10...20 {} // Error: 10 is handled twice
    17 {} // Error: 17 is handled twice
}
```

Actually, the compiler throws an error if a specific case is handled twice. Probably better to document that instead.